cmd/stolon-pgbouncer/main.go

```diff
@@ -191,6 +194,12 @@ var (
 		},
 		[]string{"keeper"},
 	)
+	storeCertificateExpirySeconds = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "stolon_store_certificate_expiry_seconds",
```

Comment:
I see this as a certificate for the stolons store, then along with everywhere else this metric is used (in the stolon components and anything else that could talk to the stolon store) you then have a single metric to look up to get a list of all components talking to the store and their certificates expiry. This in particular makes alerting easier for all the client certificates across everything communicating with the store.

WDYT?

Reply:
Hmm fair enough, that is a reasonable argument. If we subscribe to that notion (rather than enforcing that everything should be prefixed with `stolon_pgbouncer_`) then I do still think that we're inconsistent with the `store_poll_interval` and `store_last_update_seconds` metrics though.

It feels a bit like we're trading off convenience over correctness (if we wanted to write only one alert then we could get there via a recording rule); but I'm happy to go with this now in the interests of time and we can revisit it if we see it as a problem in the future.